src/VisualStudio/Core/Def/Implementation/Diagnostics/FxCopAnalyzersSuggestedActionCallback.cs

```diff
@@ -87,11 +88,16 @@ public void OnSuggestedActionExecuted(SuggestedAction action)
                 return;
             }
 
-            // Only show if the VSIX and NuGet are not installed, the info bar hasn't been shown this session,
-            // and the user is candidate based on light bulb usage.
+            // Only show if following conditions are satisfied:
+            // 1. Info bar hasn't been shown this session
+            // 2. FxCopAnalyzers VSIX is not installed 
+            // 3. FxCopAnalyzers NuGet is not installed
+            // 4. There are no unresolved analyzer references for active project and
+            // 5. User is candidate based on light bulb usage.
             if (!_infoBarChecked &&
                 !IsVsixInstalled() &&
-                !IsNuGetInstalled() &&
+                !IsNuGetInstalled(out var hasUnresolvedAnalyzerReference) &&
+                !hasUnresolvedAnalyzerReference &&
```

Comment:
❔ What happens if this check is left out? It seems like this doesn't actually change anything.

Reply:
The changes to this file are not necessary for fixing the underlying bug. This is just making this component more conservative in presence of unresolved references as it can likely lead to false positives.